custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py

```diff
@@ -21,6 +22,7 @@
 class JavaPredictor(BaseLanguagePredictor):
     JAVA_COMPONENT_ENTRY_POINT_CLASS = "com.datarobot.custom.PredictorEntryPoint"
     JAVA_COMPONENT_CLASS_NAME = "com.datarobot.custom.ScoringCode"
+    JAVA_COMPONENT_CLASS_NAME_H2O = "com.datarobot.custom.H2OPredictor"
 
```

Comment:
ideally it should be handled by map, define a map:
java_class_by_ext = {
JAR_EXTENSION: JAVA_COMPONENT_CLASS_NAME_DATAROBOT,
POJO_EXTENTION: JAVA_COMPONENT_CLASS_NAME_H2O,
MOJO_EXTENTION: JAVA_COMPONENT_CLASS_NAME_H2O
}

and the just lookup by extention

Reply:
i'll make this change.

Reply:
done